File: Apahida backend/src/main/java/com/cezar/apahida/dto/DishDTO.java
package com.cezar.apahida.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class DishDTO {
    private String name;
    private int price;
    private int stock;
    private List<String> images;
}

File: Apahida_frontend/src/app/models/dish.dto.ts
import { ImageUrlDTO } from "./ImageUrlDTO";

export class DishDTO {

  id: number = 0;
  name: string = '';
  price: number = 0;
  stock: number = 0;
  images: ImageUrlDTO[] = [];

  constructor(id: number, name: string, price: number, stock: number, images : ImageUrlDTO[]) {
    this.id = id;
    this.name = name;
    this.price = price;
    this.stock = stock;
    this.images = images;
  }

}
